server/storage/sql_models.go

```diff
@@ -50,7 +50,7 @@ func (c Change) TableName() string {
 // CreateTUFTable creates the DB table for TUFFile
 func CreateTUFTable(db gorm.DB) error {
 	// TODO: gorm
-	query := db.Set("gorm:table_options", "ENGINE=InnoDB DEFAULT CHARSET=utf8").CreateTable(&TUFFile{})
+	query := db.AutoMigrate(&TUFFile{})
```

Comment:
seems like we can still have the `Set()` call here and chain that into an `AutoMigrate`: http://jinzhu.me/gorm/database.html#migration